### PR TITLE
#BN-37 chores: setting environment variables and added dotenv to the project

### PR DESCRIPTION
**Pull request by uwdjany**
# PR Description
in this pull request ,I added dotenv to the project and set environment variables.
and I tried to configure the database, which I had converted from json to Js in order to configure a variable from the .env file.

# Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Added dotenv to the project
- [x] setting environment variables
- [x] shared variable and managed using dotenv
- [x] Moving to production switches to the correct environment
# Description of tasks that were expected to be completed and that are not yet completed
- [x] None

# How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `npm i` to install all required dependencies
2. run `npm start`
3. step 3
4. step 4...
# Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/database/config/config.js
Comment (chrisostome01) [inline src/database/config/config.js:1]: Would please consider formatting this file with prettier?

File: src/database/config/config.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      host: process.env.DATABASE_HOST,
+      dialect: "postgres"
+    }
+  
+  
+
+}
+
+
+module.exports=config;
```
Comment (chrisostome01) [inline src/database/config/config.js]: Would you try to ES6 here?
```suggestion
export default config;
```

File: src/database/config/config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (chrisostome01) [inline src/database/config/config.js]: These empty lines are not necessary. One line would be okay

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
# Great job @uwdjany 
Please consider naming the pull request title as the structure we defined for the team also I left some comments and they are mostly related to best practices please check them out too. 
Please try and integrate [cross-env](https://www.npmjs.com/package/cross-env) to update the env according to the script running.

File: src/database/config/config.js
Comment (uwdjany) [reply src/database/config/config.js:1]: check again i changed the title and prettier does not work in my vs code.

**Review by chrisostome01 — APPROVED** [review]
LGTM 🚀

**Review by kunda4 — APPROVED** [review]

**Pull request merged**